roles/srcds/files/addons/sourcemod/scripting/votescramble.sp

```diff
@@ -86,7 +95,7 @@ public Action Cmd_VoteScramble(int client, int args)
 
 public OnClientSayCommand_Post(int client, const char[] command, const char[] sArgs)
 {
-	if (strcmp(sArgs, "votescramble", false) == 0 || strcmp(sArgs, "vscramble", false) == 0 || strcmp(sArgs, "scramble", false) == 0)
+	if (strcmp(sArgs, "votescramble", false) == 0 || strcmp(sArgs, "vscramble", false) == 0 || strcmp(sArgs, "scramble", false) == 0 || strcmp(sArgs, "fuck these teams", false) == 0 )
```

Comment:
feel free to drop "fuck these teams" from the list, i just added that and "scrimblo" as a joke :P

Reply:
i'll remove "fuck these teams" (it came as part of 1.3) but re-add scrimblo because some people actually use it